README.md

```diff
@@ -134,7 +134,7 @@ $ rpm -ivh https://github.com/aquasecurity/trivy/releases/download/{TRIVY_VERSIO
 Add repository to `/etc/apt/sources.list.d`.
 
 ```
- $ sudo apt-get install wget apt-transport-https gnupg lsb-release
+$ sudo apt-get install wget apt-transport-https gnupg lsb-release
```

Comment:
I'm sorry I didn't notice this PR fixes it and I've already fixed it.
https://github.com/aquasecurity/trivy/pull/493/commits/3cbe4db80049a61f15915c0f5f4a92d470fc6420#diff-04c6e90faac2675aa89e2176d2eec7d8R137